applications/FluidDynamicsApplication/custom_elements/stokes_3D_twofluid.h

```diff
@@ -371,6 +371,38 @@ class Stokes3DTwoFluid
                 Output = 0.5*inner_prod(data.stress, strain);
             }
         }
+        else if(rVariable == EQ_STRAIN_RATE) //compute effective strain rate
+        {
+                Output = mp_constitutive_law->GetValue(rVariable, Output);
```

Comment:
I do not like this very much (for all public constitutive laws this simply uses  the GetValue from the very base class)  but I do not know any better solution